XIVComboVX/Config/PluginConfiguration.cs

```diff
@@ -412,6 +412,15 @@ public class PluginConfiguration: IPluginConfiguration {
 
 	[LucidWeavingSetting(CustomComboPreset.WhiteMageLucidWeave)]
 	public uint WhiteMageLucidWeaveManaThreshold { get; set; } = 7000;
+	
+	[ComboDetailSetting(
+		CustomComboPreset.WhiteMageDotRefresh,
+		"Debuff time threshold",
+		"When your current target's debuff only has this many seconds left, switch back to DOT",
```

Comment:
For clarity, it might be good to specify `current target's Aero/Aero2/Dia debuff` here